AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -176,13 +178,11 @@ - (void)_layoutNodes:(NSArray *)nodes ofKind:(NSString *)kind atIndexPaths:(NSAr
       dispatch_queue_t queue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
       dispatch_apply(batchCount, queue, ^(size_t i) {
         unsigned long k = j + i;
-        ASCellNodeBlock cellBlock = nodes[k];
-        ASCellNode *node = cellBlock();
+        ASIndexedNodeContext *context = contexts[k];
+        ASCellNodeBlock nodeBlock = context.nodeBlock;
+        ASCellNode *node = nodeBlock();
```

Comment:
We should probably do all this in a single-line expression so that ARC doesn't do anything (I wish its optimizations were less mysterious and I were confident it would not cause retain/release as written - perhaps it can figure this case out?